src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -59,9 +71,13 @@ public function getBlockPrefix()
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since Symfony 4.2
      */
     public function loadChoiceList($value = null)
     {
+        @trigger_error(sprintf('Method "%s" is deprecated since Symfony 4.2. Use "choice_loader" option instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
I prefer one sentence for deprecations: `... since Symfony 4.2, use ...`

Reply:
Fixed.